examples/oneapi/cpp/source/kmeans_init/kmeans_init_default_dense.cpp

```diff
@@ -29,12 +29,14 @@ int main(int argc, char const *argv[]) {
 
     const float x_compute[] = { 1.0,  1.0,  2.0,  2.0,  1.0,  2.0,  2.0,  1.0,
                                 -1.0, -1.0, -1.0, -2.0, -2.0, -1.0, -2.0, -2.0 };
-    const float x_test[]    = { 1.0, 1.0, 2.0, 2.0 };
+    const float x_test[]    = { -2.0, -2.0, 1.0, 1.0 };
 
     const auto x_compute_table = dal::homogen_table::wrap(x_compute, row_count, column_count);
-    const auto x_test_table  = dal::homogen_table::wrap(x_test, cluster_count, column_count);
+    const auto x_test_table    = dal::homogen_table::wrap(x_test, cluster_count, column_count);
 
-    const auto kmeans_init_desc = dal::kmeans_init::descriptor<>().set_cluster_count(cluster_count);
+    const auto kmeans_init_desc =
+        dal::kmeans_init::descriptor<float, dal::kmeans_init::method::plus_plus_dense>()
```

Comment:
Name of example - kmeans_init_**default_dense**, but `method::plus_plus_dense` is not currently a default

Reply:
done